mkt/ratings/forms.py

```diff
@@ -36,10 +36,8 @@ def save(self):
                 review = form.instance
                 addon = review.addon
                 if action == REVIEW_MODERATE_DELETE:
-                    review_addon = review.addon
-                    review_id = review.id
                     review.delete()
-                    mkt.log(mkt.LOG.DELETE_REVIEW, review_addon, review_id,
+                    mkt.log(mkt.LOG.DELETE_REVIEW, addon, review,
```

Comment:
I thought so, but the logging code didn't output a useful link without the object itself (not an issue before, when the object was already deleted).